src/test/java/com/google/sps/MockRQ.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps;
+
+import com.google.step.YTLounge.data.Video;
+import java.util.HashMap;
+import java.util.Map;
+
+public class MockRQ {
```

Comment:
what is RQ?

Reply:
RQ is supposed to be RetrieveQueue so that I know which class I'm mocking